public/static/app/common/tagInput.html

```diff
@@ -0,0 +1 @@
+<input type="text" ng-class="$ctrl.klass"placeholder="{{$ctrl.placeholder}}" ng-model="$ctrl.items" />
```

Comment:
Could you make the class property "class"?

Reply:
There are some other options, but no one will allow `class` (because it will be applied to parent node, which I don't want)

Reply:
I can use `directive` instead of `component`, which is less purist and could do exactly what I want, however it will be less compatible with modern Angular (in case we wan't to upgrade in the future)

Reply:
`klass` is, actually, kind of normal. React guys do similar things. It's more question of taste or code style and is not a real problem.